src/cron/schedule-google-calendar.js

```diff
@@ -0,0 +1,90 @@
+const { CronJob } = require('cron');
+const { listEvents } = require('../../calendar');
+const { EmbedBuilder } = require('discord.js');
+
+const channelId = '1306251153855610922';
+
+/**
+ * Schedules a notification to be sent when an event starts.
+ *
+ * @param {Object} event - The event object containing details about the event.
+ * @param {string} event.summary - The name or description of the event.
+ * @param {Object} event.start - The start date object for the event.
+ * @param {string} event.start.dateTime - The date and time (ISO string) when the event starts.
+ * @param {string} timeZone - The timezone to use for scheduling the notification.
+ */
+const scheduleEventNotification = async ({ client, event }) => {
+  if (
+    !client ||
+    !event ||
+    !event.start ||
+    !event.start.dateTime ||
+    !event.start.timeZone
+  ) {
+    return console.log(
+      'Error: Missing one or more required parameters (client, event, event.start.dateTime, or timeZone).'
+    );
+  }
+
+  const dateToCronExpression = (dateString) => {
+    const date = new Date(dateString);
+    const minutes = date.getMinutes();
+    const hours = date.getHours();
+    const day = date.getDate();
+    const month = date.getMonth() + 1;
+    return `${minutes} ${hours} ${day} ${month} *`;
+  };
+
+  const cronExpression = dateToCronExpression(event.start.dateTime);
+  new CronJob(
+    cronExpression,
+    async () => {
+      const currentChannel = await client.channels.cache.get(channelId);
+      const embed = new EmbedBuilder()
+        .setColor('#0099ff')
+        .setTitle('📅 Meeting reminder!')
+        .setDescription('Hello! We are meeting in **Task Planning**.')
+        .setFooter({
+          text: 'Meet remember APP',
+          iconURL:
+            'https://static.vecteezy.com/system/resources/previews/007/683/528/non_2x/calendar-calendar-icon-calendar-icon-simple-sign-calendar-symbol-free-vector.jpg',
```

Comment:
Not sure if there is an alternative to this icon?

Reply:
I decided delete it because it's not necessary